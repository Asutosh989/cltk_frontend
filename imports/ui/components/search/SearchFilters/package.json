{
  "name": "SearchFilters",
  "version": "0.0.1",
  "private": true,
  "main": "./SearchFilters.js"
}
