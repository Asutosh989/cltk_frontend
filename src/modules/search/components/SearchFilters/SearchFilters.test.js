
import React from 'react';
import { shallow } from 'enzyme';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

// component:
import SearchFilters from './SearchFilters';

describe('SearchFilters', () => {
	it('renders correctly', () => {

		const wrapper = shallow(
			<MuiThemeProvider>
				<SearchFilters
					filters={[]}
					toggleSearchTerm={() => {}}
					/>
			</MuiThemeProvider>
			);
		expect(wrapper).toBeDefined();
	});
});
