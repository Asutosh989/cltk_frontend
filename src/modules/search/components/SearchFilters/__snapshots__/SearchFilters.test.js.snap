// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchFilters renders correctly 1`] = `
<div
  className="filters"
/>
`;
